packages/frontend/src/lib/libp2p.ts

```diff
@@ -64,12 +70,15 @@ export async function startLibp2p() {
       ignoreDuplicatePublishError: true,
     }),
     identify: {
-      maxPushOutgoingStreams: 100,
-      maxInboundStreams: 100,
-      maxOutboundStreams: 100,
+      // these are set because we were seeing a lot of identify and identify push
+      // stream limits being hit
+      maxPushOutgoingStreams: 1000,
+      maxPushIncomingStreams: 1000,
+      maxInboundStreams: 1000,
+      maxOutboundStreams: 1000,
```

Comment:
I am concerned by this.  These limits are per connection, this means you're allowing 1000 identify streams to be open at the same time on each connection.

If you're seeing errors it sounds like something's not cleaning up the streams.

Maybe related: https://github.com/libp2p/js-libp2p/pull/1424

Reply:
Do what you have to to appease the demo gods though - this is more of a note to self to investigate.

Reply:
Yeah the upper bounds here are crazy, but we were hitting the limits quite